src/custom/state/price/updater.ts

```diff
@@ -89,10 +88,11 @@ export default function FeesUpdater(): null {
 
   const sellCurrency = useCurrency(sellToken)
   const buyCurrency = useCurrency(buyToken)
-  const feesMap = useAllFees({ chainId })
+  const feesMap = useAllQuotes({ chainId })
```

Comment:
nit
```suggestion
  const quotesMap = useAllQuotes({ chainId })
```
if accepted also change below